process_single_movie.py

```diff
@@ -0,0 +1,49 @@
+import os
```

Comment:
What is the difference between `process_single_movie.py` and `process_multiple_plates.py`?

To me it seems that the single_movie script is for prototyping? If so, then it probably belongs in a jupyter notebook that will enable you to explicitly describe, in great detail, everything that is going on